pkg/servercounter/counter.go

```diff
@@ -0,0 +1,14 @@
+package servercounter
```

Comment:
Please add package comments and doc comments for every public type/function: https://tip.golang.org/doc/comment

Reply:
Done!